src/odb/src/db/dbTypes.cpp

```diff
@@ -1710,4 +1710,60 @@ const char* dbDirection::getString() const
   return value;
 }
 
+dbMTermShapeType::dbMTermShapeType(const char* value)
+{
+  if (strcasecmp(value, "NONE") == 0)
+    _value = NONE;
+
+  else if (strcasecmp(value, "RING") == 0)
+    _value = RING;
+
+  else if (strcasecmp(value, "ABUTMENT") == 0)
+    _value = ABUTMENT;
+
+  else if (strcasecmp(value, "FEEDTHRU") == 0)
+    _value = FEEDTHRU;
+}
```

Comment:
There should be a final else in case of no match to avoid an uninit value (ie else _value = NONE).